src/controllers/Document.js

```diff
@@ -188,7 +189,10 @@ class DocumentController extends BaseController {
   }
 
   _search (index, collection, body = {}, options = {}) {
-    if (options.verb && options.verb.toLowerCase() === 'get') {
+    if ( this._kuzzle.protocol instanceof HttpProtocol
```

Comment:
It would be better to have a `name` property on the protocol objects

Reply:
yes thanks, I was wondering